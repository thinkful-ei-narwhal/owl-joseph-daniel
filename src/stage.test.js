import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Stage from './stage';
import STORE from './store';


describe('<Stage />', () => {
  it('smoke test', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Stage participants={STORE.participants}/>, div);
    // if something is wrong, itd fail here
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    const tree = renderer.create(<Stage participants={STORE.participants}/>).toJSON();
    expect(tree).toMatchSnapshot();Stage
  });
});