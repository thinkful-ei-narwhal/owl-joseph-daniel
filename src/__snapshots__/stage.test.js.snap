// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage /> renders this UI as expected 1`] = `
<div
  className="participant-list"
>
  <div
    className="StagePart"
  >
    <h3>
      Koren Templeton
    </h3>
    <img
      alt="Koren Templeton"
      src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
    />
  </div>
  <div
    className="StagePart"
  >
    <h3>
      Ashla Attwool
    </h3>
    <img
      alt="Ashla Attwool"
      src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
    />
  </div>
</div>
`;
